website/addons/osfstorage/views.py

```diff
@@ -265,8 +289,10 @@ def osfstorage_add_tag(file_node, **kwargs):
 def osfstorage_remove_tag(file_node, **kwargs):
     data = request.get_json()
     tag = data['tag']
+    tag_string = tag
```

Comment:
This isn't necessary. `Tag` objects are keyed on their "tag string", so `tag == Tag.load(tag)._id`, if that `Tag` exists. If not, you don't need to be logging it anyway.